plugins/GSdx/Renderers/Common/GSOsdManager.cpp

```diff
@@ -64,6 +64,7 @@ GSOsdManager::GSOsdManager() : m_atlas_h(0)
 	m_opacity               = std::max(0, std::min(theApp.GetConfigI("osd_color_opacity"), 100));
 	m_log_timeout           = std::max(2, std::min(theApp.GetConfigI("osd_log_timeout"), 10));
 	m_max_onscreen_messages = std::max(1, std::min(theApp.GetConfigI("osd_max_log_messages"), 20));
+	m_monitor_pos			= (gsosd_pos)std::max(0,std::min(theApp.GetConfigI("osd_monitor_pos"),8));
```

Comment:
space after `,`

Reply:
also you can do:
```cpp
enum class OSDPOS
{
  TOPLEFT,
  //...
  BTMRIGHT,
  MAX
}

(OSD_POS)std::max(0, std::min(theApp.GetConfigI("osd_monitor_pos"), (int)OSDPOS::MAX));
```

this way it doesn't need to be manually edited when you add/remove values.

Reply:
Spaces instead of tabs for alignment as well.